src/signature.js

```diff
@@ -130,7 +130,7 @@ export async function signedFetch(url, init = {}) {
  *
  * @returns {Promise<Response>}
  */
-function _signedFetchJSON(method = 'GET', url, init = {}) {
+function _signedFetchJSON(url, method = 'GET', init = {}) {
```

Comment:
@pburke I had to reverse the order of arguments here to get around [eslint default-param-last](https://eslint.org/docs/latest/rules/default-param-last) -- does that seem reasonable to you? I conceptually understand why you ordered things the way you did, and simultaneously looking at the docs I understand why the rule exists, and it seems like you're fairly attuned to this ecosystem.